.github/workflows/deploy-tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy tests
+
+on:
+  pull_request:
+
+jobs:
+  cancel-previous-runs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
```

Comment:
Note to myself: think about if this is necessary any more or if the concurrency global parameter replaced it